Loungeware/objects/___obj_pause/Create_0.gml

```diff
@@ -5,6 +5,12 @@ microgame_current_metadata = ___MG_MNGR.microgame_current_metadata;
 instances_deactivated = false;
 jam_id = noone;
 
+input_cooldown = 0;
```

Comment:
this is a lot of newlines you can actually just write all vars on one line to save costs on whitespace